generated-docs/FRP/Event/Mouse.md

```diff
@@ -24,4 +24,20 @@ up :: Event Int
 
 Create an `Event` which fires when a mouse button is released
 
+#### `withPosition`
+
+``` purescript
+withPosition :: forall a. Event a -> Event { value :: a, pos :: Nullable { x :: Int, y :: Int } }
```

Comment:
Maybe I can keep the original interface here if I move the setup code into `Eff` instead of in the event's subscription function.

Reply:
This turned out to be more difficult than I thought, so I'm leaving it as it is for now.